app/bundles/FormBundle/Form/Type/FormFieldSliderType.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\FormBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Custom IntegerType that supports constraints.
+ */
+final class ConstrainedIntegerType extends IntegerType
+{
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        parent::configureOptions($resolver);
+
+        $resolver->setDefined('constraints');
+        $resolver->setAllowedTypes('constraints', ['array']);
+        $resolver->setAllowedValues('constraints', function ($constraints) {
+            if (!is_array($constraints)) {
+                return false;
+            }
+
+            foreach ($constraints as $constraint) {
+                if (!$constraint instanceof Constraint) {
+                    return false;
+                }
+            }
+
+            return true;
+        });
+    }
+}
+
+/**
+ * @extends AbstractType<mixed>
+ */
+final class FormFieldSliderType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder->add('min', ConstrainedIntegerType::class, [
+            'label'      => 'mautic.form.field.form.slider_min',
+            'label_attr' => ['class' => 'control-label'],
+            'required'   => false,
+            'attr'       => ['class' => 'form-control'],
+            'data'       => $options['data']['min'] ?? 0,
+        ]);
+
+        $builder->add('max', ConstrainedIntegerType::class, [
+            'label'       => 'mautic.form.field.form.slider_max',
+            'label_attr'  => ['class' => 'control-label'],
+            'required'    => false,
+            'attr'        => ['class' => 'form-control'],
+            'data'        => $options['data']['max'] ?? 100,
+            'constraints' => [
+                new \Symfony\Component\Validator\Constraints\Expression([
+                    'expression' => 'this.getParent()["min"].getData() < value',
```

Comment:
I'd suggest to avoid having code as a string that is evaluated at some point. It'll be hard to debug and investigate. The best case is to write a custom validation class. Same for the step field.